index.html

```diff
@@ -337,11 +356,14 @@
 maintenance of pseudonymity are preserved. Note that [=pseudonymity=], on its own, is not
 sufficient to render [=data processing=] [=appropriate=].
 
-A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who, at least in the [=context=] of
-the [=processing=] being discussed, are unable to exercise sufficient
-self-determination for any consent they may provide to be receivable. This includes for
-example children, employees with respect to their employers, people in some situations of
-intellectual or psychological impairment, or refugees.
+A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who may be unable
+to exercise sufficient self-determination in the [=context=] of a specific kind of
+[=processing=]. Amongst other things, they should be treated with greater default
+privacy protections and should be considered unable to [=consent=] to the [=processing=].
+People can be vulnerable for different reasons, for example because they are children,
+are employees with respect to their employers, are facing a steep asymmetry of power,
+are people in some situations of intellectual or psychological impairment, or are
+refugees.
```

Comment:
Probably "..., are refugees, etc." too avoid anyone thinking this is a complete list.